src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/ScoreboardElements.kt

```diff
@@ -0,0 +1,633 @@
+            listOf("<hidden>" to HorizontalAlignment.LEFT)
+        }
+    }
+
+    fun isVisible(): Boolean {
+        if (!informationFilteringConfig.hideIrrelevantLines) return true
+        return showWhen()
+    }
+}
+
+
+private fun getTitleDisplayPair() = when (displayConfig.titleAndFooter.useHypixelTitleAnimation) {
+    true -> listOf(ScoreboardData.objectiveTitle to getTitleAndFooterAlignment())
+    false -> listOf(
+        displayConfig.titleAndFooter.customTitle.get().toString()
+            .replace("&", "§") to getTitleAndFooterAlignment()
+    )
+}
+
+private fun getProfileDisplayPair() =
+    listOf(CustomScoreboardUtils.getProfileTypeSymbol() + HypixelData.profileName.firstLetterUppercase() to HorizontalAlignment.LEFT)
+
+private fun getPurseDisplayPair(): List<ScoreboardElementType> {
+    var purse = PurseAPI.currentPurse.toString().formatNum()
+
+    val earned = getGroupFromPattern(ScoreboardData.sidebarLinesFormatted, PurseAPI.coinsPattern, "earned")
+
+    if (earned != "0") {
+        purse += " §7(§e+$earned§7)§6"
+    }
+
+    return when {
+        informationFilteringConfig.hideEmptyLines && purse == "0" -> listOf("<hidden>")
+        displayConfig.displayNumbersFirst -> listOf("§6$purse Purse")
+        else -> listOf("Purse: §6$purse")
+    }.map { it to HorizontalAlignment.LEFT }
+}
+
+private fun getPurseShowWhen() = !inAnyIsland(IslandType.THE_RIFT)
+
+private fun getMotesDisplayPair(): List<ScoreboardElementType> {
+    val motes = getGroupFromPattern(ScoreboardData.sidebarLinesFormatted, ScoreboardPattern.motesPattern, "motes")
+        .formatNum()
+
+    return when {
+        informationFilteringConfig.hideEmptyLines && motes == "0" -> listOf("<hidden>")
+        displayConfig.displayNumbersFirst -> listOf("§d$motes Motes")
+        else -> listOf("Motes: §d$motes")
+    }.map { it to HorizontalAlignment.LEFT }
+}
+
+private fun getMotesShowWhen() = inAnyIsland(IslandType.THE_RIFT)
+
+private fun getBankDisplayPair(): List<ScoreboardElementType> {
+    val bank = getGroupFromPattern(TabListData.getTabList(), ScoreboardPattern.bankPattern, "bank")
+
+    return when {
+        informationFilteringConfig.hideEmptyLines && bank == "0" -> listOf("<hidden>")
+        displayConfig.displayNumbersFirst -> listOf("§6$bank Bank")
+        else -> listOf("Bank: §6$bank")
+    }.map { it to HorizontalAlignment.LEFT }
+}
+
+private fun getBankShowWhen() = !inAnyIsland(IslandType.THE_RIFT)
+
+private fun getBitsDisplayPair(): List<ScoreboardElementType> {
+    val bits = if (BitsAPI.bits == -1) {
+        "0"
+    } else {
+        BitsAPI.bits.formatNum()
+    }
+    val bitsToClaim = if (BitsAPI.bitsToClaim == -1) {
+        "§cOpen Sbmenu§b"
+    } else if (BitsAPI.bitsToClaim < -1) {
+        "0"
+    } else {
+        BitsAPI.bitsToClaim.formatNum()
+    }
+
+    val bitsDisplay = when {
+        informationFilteringConfig.hideEmptyLines && bits == "0" -> listOf("<hidden>")
```

Comment:
this is behaves unexpectedly if showUnclaimedBits is turned on. Please change up the case like the others below

Reply:
I dont think I see how this can behave unexpectedly.
Maybe bc hides the line when 0 bits but bits to claim? Im not sure

Reply:
Unexpected for the user. If you have 0 bits but 2000 bits to claim, I want to see the bits to claim.